interface-definitions/include/interface-mirror.xml.i

```diff
@@ -1,11 +1,31 @@
 <!-- included start from interface-mirror.xml.i -->
-<leafNode name="mirror">
+<tagNode name="mirror">
```

Comment:
Now as this is a `tagNode` can I mirror to multiple interfaces?

Reply:
Yes, it can be designed according to schedule